src/index.js

```diff
@@ -0,0 +1,2 @@
+import './styles/variables.scss';
```

Comment:
maybe the .scss doesn't have to be imported here?

(unless there's something in this javascript file that uses CSS declarations)

Reply:
It's a build thing.  I'd have to explain